content/doc/pipeline/tour/running-multiple-steps.adoc

```diff
@@ -88,6 +88,11 @@ There are some powerful steps that "wrap" other steps which can easily solve
 problems like retrying (`retry`) steps until successful or exiting if a
 step takes too long (`timeout`).
 
+When a step cannot be completed, timeouts help the controller avoid wasting resources.
+This video reviews the process of setting up both stage and global timeouts.
+
+video::OChOtpK0fUE[youtube,width=800,height=420]
```

Comment:
.How do I set a timeout in a Jenkins Pipeline
```suggestion
video::OChOtpK0fUE[youtube,width=800,height=420]
```

Reply:
Hey @cari-mason thanks for the review, and I'm checking in with @darinpope on the captioning aspect of our videos. I'll be sure to create a further ticket once we've been able to connect on next steps!